runtime/hydra-dx/src/tests.rs

```diff
@@ -15,9 +14,9 @@ fn full_block_cost() {
 
 	let max_weight = BlockWeights::get().get(DispatchClass::Normal).max_total.unwrap_or(1);
 	let weight_fee = WeightToFee::calc(&max_weight);
-	assert_eq!(weight_fee, 19149775500000);
+	assert_eq!(weight_fee, 120000000000000);
```

Comment:
does this mean that we're basically increasing-price 10 fold?

Reply:
the difference is because of removal of extra weight from ExtrinsicBaseWeight.

If i  add back the extra weight - it is the same result 19149775500000.

Which is kind of surprising 😕